test/test_issues/test_issue1808.py

```diff
@@ -0,0 +1,44 @@
+from rdflib import Graph
+from rdflib.term import BNode, URIRef, rdflib_skolem_genid
```

Comment:
Ideally there should be tests for this in a more general test suite, as I think in the long term we have to eliminate all the issue specific tests.

Reply:
Fine for now though.